cmd/monitor/monitor.go

```diff
@@ -92,6 +97,8 @@ func (m *AppCreateAvailabilityMonitor) Run() error {
 	wait.Until(func() {
 		glog.V(2).Infof("starting a test")
 
+		m.deleteMonitorProjects()
```

Comment:
If this returns an error, we should return early and try again next interval. Otherwise we'll continue to accumulate namespaces in the failure condition the patch is meant to address.